### DISCUSSION TITLE
Improve application performance

### DISCUSSION THREAD

**Discussion by NguyenDuyHieu987** [nuxt/nuxt#25215] [Questions]
Hello!
I have a [Project](https://github.com/NguyenDuyHieu987/Movie-Nuxt-App) with Nuxt 3 and [Here](https://phimhay247z.org) is the website link of that project.
I don't know why my website is so slow and laggy.
### My `nuxi analyze`

Nitro
![image](https://github.com/nuxt/nuxt/assets/104075342/ebeae973-7521-43bc-a5c6-4af330897ba7)
Client
![image](https://github.com/nuxt/nuxt/assets/104075342/e245d9c7-9f0b-4be9-a56e-d38acfe32d21)

Does anyone know why and how to solve it?
Thanks a lot!

**Comment by xjccc** [comment]
you can watch some videos of `Alexander Lichter`

[Reduce the size of your payload](https://www.youtube.com/watch?v=laRJNkG_wls&t=336s)

[The Lazy component prefix and when to use it](https://www.youtube.com/watch?v=YbAQC1yetUM)

and trying to do something
